tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -464,6 +464,38 @@ TEST(RGBToYUVTest, HighBitDepthGrayToYUV420) {
   }
 }
 
+TEST(RGBToYUVTest, 8BitGrayRoundTripWithLift) {
+  // 2x2 12-bit temporary image.
+  static constexpr uint8_t gray[4] = {4, 3, 2, 1};
```

Comment:
Nit: Move this line after the comment at line 472, `2x2 8-bit original image.` Alternatively move this line before the comment at line 468, `2x2 12-bit temporary image.`

At the current location, this is confusing because the comment at line 468 says `12-bit` but this array is 8 bits.

Reply:
Done